app/controllers/presentation_orders_controller.rb

```diff
@@ -0,0 +1,47 @@
+class PresentationOrdersController < ApplicationController
+  before_action :require_active_current_user
+  before_action :require_privilege, only: :create
+  before_action :set_meeting
+
+  def index
+  end
+
+  def create
```

Comment:
Assignment Branch Condition size for create is too high. [23.85/20]<br>Method has too many lines. [21/10]